src/index.js

```diff
@@ -14,35 +17,54 @@ class RtlCssPlugin {
       filename: '[name].rtl.css',
     };
   }
+  
+  processAssets(compilation, callback) {
+    // Explore each chunk (build output):
+    compilation.chunks.forEach(chunk => {
+      // Explore each asset filename generated by the chunk:
+      const files = isWebpack5 ? [...chunk.files] : chunk.files;
```

Comment:
What is the type of chunk?
Why do you need to spread it into an array?

Reply:
Oh, forgot to explain that with the Github UI. Webpack 5 changed this from an array to a `Set`. It's easier to turn in into an array for filtering/mapping it bellow.

Reply:
I assumed, but wanted to be sure :) Thanks for explaning